scripts/zones/Grauberg_[S]/npcs/34.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Grauberg [S]
+--  NPC:
+--  Quest - Seeing Blood-red
+-- pos -283 -55 -97
+-----------------------------------
+local ID = require("scripts/zones/Grauberg_[S]/IDs")
+require("scripts/globals/quests")
+require("scripts/globals/keyitems")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    if player:getCharVar("SeeingBloodRed") == 3 then
+        player:startEvent(14)
+    end
+end
+
+function onEventUpdate(player,csid,option)
+
+end
+
+function onEventFinish(player,csid,option)
+    if csid == 14 then
+      --player:delKeyItem(tpz.ki.PORTING_MAGIC_TRANSCRIPT)
+      --TODO: Teleport player to Ruhotz Silvermines
+      --AFTER BCNM:player:setCharVar("SeeingBloodRed",5)
```

Comment:
At the end of the BCNM fight, this line should execute. I put it here because the BCNM isn't scripted. Once this is set, the quest will continue as I've written it.